deployment/network-operator/charts/sriov-network-operator/values.yaml

```diff
@@ -0,0 +1,35 @@
+# Default values for sriov-network-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+operator:
+  tolerations:
+    - key: "node-role.kubernetes.io/master"
+      operator: "Equal"
+      value: ""
+      effect: "NoSchedule"
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  nameOverride: ""
+  fullnameOverride: ""
+  # tag, if defined will use the given image tag, else Chart.AppVersion will be used
+  # tag
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
```

Comment:
Non-related comments

Reply:
These comments are auto generated by helm when it creates a package template, from my POV these can be left as is for the future as to ensure that resources are not explicitly specified